magnum_cluster_api/resources.py

```diff
@@ -1143,11 +1144,14 @@ def labels(self) -> dict:
         ccm_version = utils.get_cluster_label(
             self.cluster, "cloud_provider_tag", CLOUD_PROVIDER_TAG
         )
-        cni_verison = utils.get_cluster_label(self.cluster, "calico_tag", CALICO_TAG)
-
+        cni_version = utils.get_cluster_label(self.cluster, "calico_tag", CALICO_TAG)
+        calico_ipv4pool = utils.get_cluster_label(
+            self.cluster, "calico_ipv4pool", CALICO_IPV4_POOL
+        ) 
```

Comment:
we can drop this, since it doesnt really affect the deployment of the 'cluster addon'

Reply:
removed it https://github.com/vexxhost/magnum-cluster-api/commit/84f1fb3d3870ed8be5f5f436dc01f7ce8ff69ad9